File: app/src/main/java/com/zr/demo/ToastUtils.java
package com.zr.demo;

import android.content.Context;
import android.widget.Toast;

/**
 * Created by zr on 2017/2/5.
 */

public class ToastUtils {

    /** Show short Toast.
     *
     *  @see Toast#makeText(Context, CharSequence, int)
     *  @see Toast#LENGTH_SHORT
     */
    public static void showShort(Context context, CharSequence text) {
        Toast.makeText(context, text, Toast.LENGTH_SHORT).show();
    }
}

File: app/src/main/java/com/zr/demo/ToastUtils.kt
package com.zr.demo

import android.content.Context
import android.widget.Toast

/**
 * Created by zr on 2017/2/5.
 */
object ToastUtils {
    /** Show short Toast.
     *
     * @see Toast.makeText
     * @see Toast.LENGTH_SHORT
     */
    fun showShort(context: Context?, text: CharSequence?) {
        Toast.makeText(context, text, Toast.LENGTH_SHORT).show()
    }
}